src/stores/app/index.js

```diff
@@ -257,26 +286,40 @@ export default class AppStore extends Store {
   };
 
   @action
-  modify = async (params = {}) => {
+  modify = async (params = {}, hasNote) => {
     this.isLoading = true;
     this.createResult = await this.request.patch('apps', params);
     this.isLoading = false;
 
-    if (get(this.createResult, 'app_id')) {
+    if (hasNote && get(this.createResult, 'app_id')) {
       this.info('应用信息保存成功');
     }
   };
 
   @action
   modifyApp = async e => {
-    e.preventDefault();
+    if (e) {
```

Comment:
what if e is not event object ?

Reply:
The app version submit will used this Info page, the LayoutStepper Component next step will used this modifyApp.